packages/headless/src/app/commerce-engine/commerce-engine-configuration.ts

```diff
@@ -0,0 +1,22 @@
+import {Schema} from '@coveo/bueno';
+import {
+  EngineConfiguration,
+  engineConfigurationDefinitions,
+  getSampleEngineConfiguration,
+} from '../engine-configuration';
+
+/**
+ * The commerce engine configuration options.
+ */
+export interface CommerceEngineConfiguration extends EngineConfiguration {}
+
+export const commerceEngineConfigurationSchema =
+  new Schema<CommerceEngineConfiguration>({
+    ...engineConfigurationDefinitions,
+  });
+
+export function getSampleCommerceEngineConfiguration(): CommerceEngineConfiguration {
+  return {
+    ...getSampleEngineConfiguration(),
```

Comment:
I would suggest using a test "commerce" organization for this. 

`fashioncoveodemocomgzh7iep8` could be a good one.

`searchuisamples` is just too much random sources and content not linked to commerce at all.